test/integration/main_test.go

```diff
@@ -231,6 +241,9 @@ contacts:
 					t.Errorf("iamPolicies is blank in resourcemapping.annotations")
 				}
 			}
+			if !strings.Contains(gotProcessError, tc.wantProcessErrSubStr) {
```

Comment:
Use https://github.com/abcxyz/pkg/blob/683ea9d2f9bd3dcd89ab76f4a2fedfda17a4d977/testutil/error.go#L25

Reply:
I think this are two strings, why do we want to wrap it two Error and compare?

Reply:
I thought the first one is Error. Then no need to change it.